cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html

```diff
@@ -93,3 +93,11 @@ <h4 class="supplement-title">Preguntas similares</h4>
         </span>
     </div><!-- .share -->
 {% endblock %}
+
+{% block page_js %}
+<script type="text/javascript">
+    var answerID = '{{ answer_id }}';
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/pull/3039/commits/b87358e2217322f64d9ffdaacc166f0803502a16. I'm using an empty div to pass the data for now, but I'll think about whether there's a better place for it.